models/Document/Editable.php

```diff
@@ -491,7 +491,7 @@ public function __toString()
                 // the __toString method isn't allowed to throw exceptions
```

Comment:
We could just remove the try/catch, since `__toString` *is* allwoed to throw exceptions since PHP 7.4: https://wiki.php.net/rfc/tostring_exceptions